purchase_request_order_approved/README.rst

```diff
@@ -0,0 +1,76 @@
+===============================
```

Comment:
Old version of file.
If you don't want to generate the README file, you can just remove it, and the ocabot will generate for you.